### PR TITLE
supporting publish emacs-libvterm to Melpa.

### PR DESCRIPTION
**Pull request by jixiuf**
make sure `package-lint` and  `checkdoc` is happy with the docstrings.
#53 
melpa/melpa#6133

### CODE REVIEW

File: vterm.el [hunk 1]
```diff
@@ -219,6 +227,8 @@ If nil, never delay")
     (vterm--delayed-redraw (current-buffer))))
 
 (defun vterm--delayed-redraw(buffer)
+  "Redraw the termnial buffer .
```
Comment (ericsolomon) [inline vterm.el]: terminal*

File: vterm.el [hunk 2]
```diff
@@ -206,11 +213,12 @@ for different shell. "
...
 
 (defvar vterm-timer-delay 0.01
-  "Delay for refreshing the terminal buffer after receiving updates from
-libvterm. Improves performance when receiving large bursts of data.
+  "Delay for refreshing the buffer after receiving updates from libvterm.
+Improves performance when receiving large bursts of data.
 If nil, never delay")
 
 (defun vterm--invalidate()
+  "The terminal buffer is invalidated,the buffer need redrawing."
```
Comment (ericsolomon) [inline vterm.el]: space after comma and need -> needs

Comment (jixiuf) [reply vterm.el]: thanks, fixed

File: vterm.el (hunk 1 above)
Comment (jixiuf) [reply vterm.el]: thanks, fixed

**Pull request merged**